.github/workflows/main.yml

```diff
@@ -125,7 +125,7 @@ jobs:
           key: mvn-liquibase-${{ github.run_id }}-${{ github.run_attempt }}
 
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I'd recommend making this a pinned version instead of `latest` due to the fact that titan has to precompile zfs releases based on the kernel and as long as we stay within a major version branch this can be automated. For example, we currently check nighly for GHA updates to the 18.04 kernel. The current choices are 22.04 or 20.04 (which ironically is `latest` not 22.04). Once a major version is pinned, I can make an update to the titan nightly jobs to start precompiling for this kernel variant.

Reply:
Spoke with Kristyl, we decided to try go straight to 22.04. It works (except titan platforms, what is expected). If @mcred can make titan work with that we can keep it, otherwise go back to 20.04.

Reply:
@kristyldatical @KushnirykOleh After working on this for a few days I have finally come the conclusion that 20.04 is the right decision. I'll give more details in a reply below.